### PR TITLE
Updates v1.31 hugo.toml for release v1.32

### PR DESCRIPTION
**Pull request by chanieljdan**
This PR updates the hugo.toml for v1.31 ahead of the v1.32 release.

Base branch will be updated from main to release-1.31 once that branch exists.

/hold for v1.32 release day

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -143,12 +143,12 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
...
 showedit = true
 
-latest = "v1.31"
+latest = "v1.32"
 
 version = "v1.31"
-githubbranch = "main"
-docsbranch = "main"
-deprecated = false
+githubbranch = "1.31.4"
```
Comment (reylejano) [inline hugo.toml:149]:
```suggestion
githubbranch = "1.31.3"
```

File: hugo.toml [hunk 2]
```diff
@@ -143,12 +143,12 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
...
+docsbranch = "release-1.32"
```
Comment (reylejano) [inline hugo.toml:150]:
```suggestion
docsbranch = "release-1.31"
```

**Review by Arhell — COMMENTED** [review]
lgtm

**Comment by reylejano** [comment]
/hold 
this should go to the release-1.31 branch

**Comment by reylejano** [comment]
base branch changed to the release-1.31 branch
/unhold

**Pull request merged**